src/components/organisms/UiPhoneNumber/UiPhoneNumber.vue

```diff
@@ -0,0 +1,384 @@
+<template>
+  <section
+    v-if="!isLoading"
+    class="ui-phone-number"
+  >
+    <fieldset class="ui-phone-number__fields">
+      <slot
+        name="legend"
+        v-bind="{ label: translation.phoneNumberLabel }"
```

Comment:
```suggestion
        v-bind="{ translation }"
```

Reply:
Q: Why bind the whole object if in this `<slot>`'s scope, the only property used is `label`? 🤔 :D